backtrace-library/src/main/java/backtraceio/library/common/ApplicationHelper.java

```diff
@@ -0,0 +1,74 @@
+package backtraceio.library.common;
```

Comment:
No unit tests for this file

Reply:
Testing getters on app context doesn't make sense. We have tests on the higher level that verify if those information are available

Reply:
1. But here you can test logic around Application information cache not only getters
2. Can you point me to place where we have any asserts in tests for these 3 fields?

Reply:
working on tests - I was fixing the part in the metrics tests that confirms the application name and version are defined. I cannot find them however its hard to manage metrics tests now. I will write dedicated tests for this